components/cli/vendor/github.com/docker/docker/client/session/filesync/diffcopy.go

```diff
@@ -9,9 +9,10 @@ import (
 	"github.com/tonistiigi/fsutil"
 )
 
-func sendDiffCopy(stream grpc.Stream, dir string, excludes []string, progress progressCb) error {
+func sendDiffCopy(stream grpc.Stream, dir string, includes, excludes []string, progress progressCb) error {
```

Comment:
Brought in with this PR:
* https://github.com/moby/moby/pull/33859 Add path filtering to build session client